schemas/dbt_project.json

```diff
@@ -215,6 +218,35 @@
             },
             "additionalProperties": false
         },
+        "metric_configs": {
+            "title": "Metric configs",
+            "type": "object",
+            "description": "Configurations set in the dbt_project.yml file will apply to all metrics that don't have a more specific configuration set.",
+            "properties": {
+                "+enabled": {
```

Comment:
Can you sort this block alphabetically pls (ie all the pluses together then the plusless ones)?